### ISSUE TITLE
visitor_uid when created leads not support

### ISSUE THREAD

**Issue by bigperson** [ufee/amoapi#2]
See: https://www.amocrm.ru/developers/content/digital_pipeline/site_visit
POST /api/v2/leads
```json
{
    "add" : [
          {
              "name" : "Заявка с сайта" ,
              "status_id" : 142 ,
              "visitor_uid" : "12345678-52d2-44c2-9e16-ba0052d9f6d6"
          }
     ]
}
```
Now: Exception, Invalid Lead field: visitor_uid
RFC:
```php
$lead = $amo->leads()->create();
$lead->visitor_uid = $visitorUid; //or $lead->setVisitor($visitorUid);
$lead->save(); //or $lead->save($visitorUid);
```

**Comment by ufee** [comment]
Fixed.

**Comment by ufee** [comment]
Fix#2

**Comment by ufee** [comment]
[https://github.com/ufee/amoapi/commit/743801b51c948cc69b3d0b8b091ef447017866b2](url)
